content/en/user-manual/publishing/mobile/gonative.md

```diff
@@ -0,0 +1,15 @@
+---
+title: GoNative
+layout: usermanual-page.hbs
+position: 2
+---
+
+[GoNative][gonative] is a paid third party service that can take a URL and create a native mobile application that uses a native WebView to run in the mobile in a few clicks.
+
+As it is using a WebView, the user has to be online to use the app and you will have to host your PlayCanvas application somewhere such as [AWS Web Hosting services][aws-webhost].
+
+GoNative also provides simple integration of mobile native plugins that can be accessed from your PlayCanvas application such as Google AdMob and In-App Purchases. A full list of [plugins can be found here][plugin-list].
+
+[gonative]: https://gonative.io/
+[aws-webhost]: https://aws.amazon.com/websites/
+[plugin-list]: https://gonative.io/plugins
```

Comment:
Link style across the repo has been to use an ordered numbered list. Did you want to change this?

Reply:
Over the last few months I've changed new links to use this style as it's more maintainable and easier to read/reuse than numbered lists. For example, if wanted to reuse a link, using a number list means that I would have to look either where the previous usage was in the text to find the number or look at the URLs to work out the number at the end.

Whereas using a text link, it's just easier to parse and search for me. Plus auto completion in VSCode means that I don't always have to refer to the end of the doc to find the link text/number.

Reply:
I think there's pros and cons to both conventions. I guess I'm just keen to avoid mixed styles all over the place. However, I don't have massively strong feeling about it right now...

Reply:
Makes, sense. Perhaps they should at least be consistent to the page?

Reply:
Sure. BTW, the [CONTRIBUTING](https://github.com/playcanvas/developer.playcanvas.com/blob/dev/.github/CONTRIBUTING.md#hyperlinks) guide dictates that a numbering system should be used. So if you want to change things, the guide should be updated.